### PR TITLE
Sync gh-pages: Use Twitter Bootstrap 4 for Pico default theme.

### PR DESCRIPTION
**Pull request by theshka**
This PR syncs the doc changes from `feature/bootstrap-theme` #297

### CODE REVIEW

File: _docs/contribute.md [hunk 1]
```diff
@@ -4,15 +4,16 @@ toc:
...
 
-1. Plugins & Themes: You're a plugin developer or theme designer? We love you guys! You can find tons of information about how to develop plugins and themes at [{{ site.base_url }}/plugin-dev.html][PluginDocs]. If you have created a plugin or theme, please add it to our [Wiki][], either on the [plugins][WikiPlugins] or [themes page][WikiThemes]. Doing so, we may select and promote your plugin or theme on [our website][OfficialPlugins] as officially supported!
+1. Plugins & Themes: You're a plugin developer or theme designer? We love you guys! You can find tons of information about how to develop plugins and themes at http://picocms.org/plugin-dev.html. If you have created a plugin or theme, please add it to our [Wiki][], either on the [plugins][WikiPlugins] or [themes page][WikiThemes]. Doing so, we may select and promote your plugin or theme on [our website][OfficialPlugins] as officially supported!
```
Comment (PhrozenByte) [inline _docs/contribute.md]: There's a reason why this is configurable :wink:

File: _docs/creating-content.md [hunk 2]
```diff
@@ -7,7 +7,8 @@ toc:
 nav: 4
 ---
 
-## Creating Content
+Creating Content
+-------
```
Comment (PhrozenByte) [inline _docs/creating-content.md]: For the sake of readability the heading syntax should remain consistent

File: _docs/creating-content.md [hunk 3]
```diff
@@ -76,7 +77,7 @@ There are also certain variables that you can use in your text files:
 
 ### Blogging
 
-Pico is not blogging software - but makes it very easy for you to use it as a blog. You can find many plugins out there implementing typical blogging features like authentication, tagging, pagination and social plugins. See the below Plugins section for details.
+Pico is not blogging software - but makes it very easy for you to use it as a blog. We even include a simple blog layout in our default template! However, you can find many plugins out there implementing typical blogging features like authentication, tagging, pagination and social plugins. See the below Plugins section for details.
```
Comment (PhrozenByte) [inline _docs/creating-content.md]: If we want to support Blogging officially, the section should be removed completely.

File: _docs/getting-help.md [hunk 4]
```diff
@@ -8,18 +8,19 @@ toc:
 nav: 7
 ---
 
-## Getting Help
+Getting Help
+------------
 
-### Getting Help as a user
-If you want to get started using Pico, please refer to the [user docs][HelpUserDocs] (you're reading them right now!). Please read the [upgrade notes][HelpUpgrade] if you want to upgrade from Pico 0.8 or 0.9 to Pico 1.0. You can find officially supported plugins and themes on [our website][OfficialPlugins]. A greater choice of third-party plugins and themes can be found in our [Wiki][] on the [plugins][WikiPlugins] or [themes][WikiThemes] pages respectively. If you want to create your own plugin or theme, please refer to the ["Getting Help as a developer" section](#getting-help-as-a-developer) below.
+### ... as a user
```
Comment (PhrozenByte) [inline _docs/getting-help.md]: Is there any reason to shorten this title? Sure, makes sense for the ToC, but in continues text it rather impedes readability.

File: _docs/getting-help.md [hunk 5]
```diff
@@ -8,18 +8,19 @@ toc:
...
+If you want to get started using Pico, please refer to our [user docs][HelpUserDocs]. Please read the [upgrade notes][HelpUpgrade] if you want to upgrade from Pico 0.8 or 0.9 to Pico 1.0. You can find officially supported plugins and themes on [our website][OfficialPlugins]. A greater choice of third-party plugins and themes can be found in our [Wiki][] on the [plugins][WikiPlugins] or [themes][WikiThemes] pages respectively. If you want to create your own plugin or theme, please refer to the "Getting Help as a developer" section below.
 
-### Getting Help as a developer
-If you're a developer, please refer to the ["Contribute" section](#contribute) below and our [contribution guidelines][ContributionGuidelines]. To get you started with creating a plugin or theme, please read the [dev docs on our website][HelpDevDocs].
+### ... as a developer
+If you're a developer, please refer to the "Contributing" section below and our [contribution guidelines][ContributionGuidelines]. To get you started with creating a plugin or theme, please read the [dev docs on our website][HelpDevDocs].
```
Comment (PhrozenByte) [inline _docs/getting-help.md]: What's the reason for removing this link?

File: _docs/getting-help.md [hunk 6]
```diff
@@ -8,18 +8,19 @@ toc:
...
 
-### You still need help or experience a problem with Pico?
+### ... still need help with Pico?
 When the docs can't answer your question or when you're experiencing problems with Pico, please don't hesitate to create a new [Issue][Issues] on GitHub. Concerning problems with plugins or themes, please refer to the website of the developer of this plugin or theme.
 
-**Before creating a new Issue,** please make sure the problem wasn't reported yet using [GitHubs search engine][IssuesSearch]. Please describe your issue as clear as possible and always include steps to reproduce the problem.
+> **Before creating a new Issue,** please make sure the problem wasn't reported yet using [GitHubs search engine][IssuesSearch]. Please describe your issue as clear as possible and always include steps to reproduce the problem.
```
Comment (PhrozenByte) [inline _docs/getting-help.md]: Bold text really should be enough highlighting :wink:

File: _docs/install.md [hunk 7]
```diff
@@ -7,28 +7,33 @@ toc:
...
 ---
 
-## Install
+Install
+-------
 
 You can install Pico either using a pre-bundled release or with composer. Pico is also available on [Packagist.org][] and may be included in other projects via `composer require picocms/pico`. Pico requires PHP 5.3+
 
-### Using a pre-bundled release
+### Using a pre-bundled release - for users
```
Comment (PhrozenByte) [inline _docs/install.md]: No! The `composer` setup is not just for developers. There's no change necessary in this section...

File: _docs/creating-content.md (hunk 2 above)
Comment (theshka) [inline _docs/creating-content.md]: That is true, so why does the README have different styled headings? :confused:

File: _docs/contribute.md (hunk 1 above)
Comment (theshka) [inline _docs/contribute.md]: Oops! Copy pasta :frowning:  I'll fix that...

File: _docs/run.md [hunk 8]
```diff
@@ -4,21 +4,24 @@ toc:
...
+    ```
 
-#### Step 2
-Start PHPs built-in web server:
-<pre><code>$ php -S 127.0.0.1:8080</code></pre>
+3. Access Pico from [http://localhost:8080](http://localhost:8080).
 
-#### Step 3
-Access Pico from http://localhost:8080.
+> Please note that PHPs built-in web server is for *development* and *testing* purposes *only!*
```
Comment (PhrozenByte) [inline _docs/run.md]: This obviously is no `<blockquote>`... :wink: This highlighting isn't necessary at all, the old version was just fine.

File: _docs/install.md (hunk 7 above)
Comment (theshka) [inline _docs/install.md]: Currently: 


Using a pre-bundled release

__Just download the latest Pico release and upload all files to the httpdocs directory (e.g. /var/www/html) of your server.__

Composer

Step 1 - for users

__Download the source code of Pico's latest release, upload all files to the httpdocs directory (e.g. /var/www/html) of your server and navigate to the upload directory using a shell.__

Step 1 - for developers

Open a shell and navigate to the desired install directory of Pico within the httpdocs directory (e.g. /var/www/html) of your server. You can now clone Pico's Git repository as follows:


Why is the "for users" instructions there twice?

File: _docs/creating-content.md (hunk 3 above)
Comment (theshka) [inline _docs/creating-content.md]: Even though I'd like to provide a blog example, it needs to be stated "Pico is not blogging software - but makes it very easy for you to use it as a blog." I don't think removing the section completely is necessary.

File: _docs/install.md (hunk 7 above)
Comment (PhrozenByte) [inline _docs/install.md]: Users can make their own choice, some people prefer `composer`, others pre-bundled releases and some people (even non-developers) want to use `git` (the latter is a little more complicated and requires some knowledge in `git`, but explaining this is out of Pico's scope, thus it's not part of the docs). The sections don't build on one another, they're completely independent.

File: _docs/creating-content.md (hunk 3 above)
Comment (PhrozenByte) [inline _docs/creating-content.md]: The point I want to make is not this particular paragraph. It just makes no sense that we have a step-by-step-tutorial below, what describes how to use Pico as a blogging software, if this is already part of the default theme.

**Pull request closed without merging**